### ISSUE TITLE
[Bug] Clicking on request result in runner cause Bruno app to be completely empty

### ISSUE THREAD

**Issue by martinsefcik** [usebruno/bruno#699]
After clicking on request result in Runner tab whole Bruno application changes to empty and I have to restart it to be able continue working with it.

![Recording 2023-10-20 140728 (1)](https://github.com/usebruno/bruno/assets/1073299/78697869-dbc2-4bfa-a46b-48d2dfdf4824)

Affected version: 0.26.0 

It looks like this bug was introduced in #622

**Comment by Its-treason** [comment]
Yep, looks like I missed that one in my PR, Thanks for the quick fix.

We also should look into having some kind of "error boundary" to show an error message instead of a blank screen.

**Comment by helloanoop** [comment]
> We already have an [Error Boundary](https://github.com/usebruno/bruno/blob/main/packages/bruno-app/src/pages/ErrorBoundary/index.js), but for some reason it's not working right, in the release build. After building Bruno locally, the Error Boundary Works:

Yeah, its quite strange. In the production build, the error boundary works at some places, but doesn't at others.

Closing this ticket as the fix has been delivered by @martinsefcik in `v0.27.0`
